### ISSUE TITLE
System load time takes long

### ISSUE THREAD

**Issue by zarathustra19** [systemd/systemd#15828]
**slow boot up**

Operating System: Fedora 32
Kernel Version: 5.6.11-300.fc32.x86_64
Processors: 4 × AMD A10-7300 Radeon R6, 10 Compute Cores 4C+6G
Memory: 6.8 GiB of RAM
Disk: SSD 120GB

**Boot time:**

`systemd-analyze Startup finished in 3.802s (firmware) + 6.806s (loader) + 4.789s (kernel) + 10.325s (initrd) + 1min 30.980s (userspace) = 1min 56.702s graphical.target reached after 1min 30.908s in userspace`

**Systemd-analyse blame**

`systemd-analyze blame 2min 6.007s dnf-makecache.service > 1min 16.773s systemd-udev-settle.service > 47.310s lvm2-monitor.service > 7.881s NetworkManager-wait-online.service > 7.089s upower.service > 6.198s systemd-rfkill.service > 5.982s dracut-initqueue.service > 4.104s unbound-anchor.service > 3.983s systemd-logind.service > 3.207s systemd-machined.service > 2.732s systemd-homed.service > 2.180s initrd-switch-root.service > 1.702s systemd-backlight@backlight:radeon_bl0.service > 1.229s systemd-udevd.service > 1.169s udisks2.service > 1.046s systemd-journald.service > 1.017s sssd.service`

**journalctl -b -u systemd-udev-settle**

```
journalctl -b -u systemd-udev-settle
-- Logs begin at Mon 2020-05-04 22:04:21 +04, end at Sat 2020-05-16 00:15:58 +0>
May 15 21:29:29 localhost.localdomain udevadm[542]: systemd-udev-settle.service>
May 15 21:30:39 localhost.localdomain systemd[1]: Finished udev Wait for Comple>
```

Waiting for solving that issues from systemd maintainers teams.

any recommendations be appreciated.

**Comment by boucman** [comment]
dnf-makecache took more than a minute to start... 
So this seems distro- specific to me...

**Comment by zarathustra19** [comment]
> dnf-makecache took more than a minute to start...
> So this seems distro- specific to me...
one chance is it will solved from distro systemd maintainers team probably right?

If any news or solve method announced please reply.

**Comment by poettering** [comment]
Please contact your distro about this.

Any distro that ends up pulling in systemd-udev-settle.service is broken in 2020. It's storage stuff that does this, and it needs fixing since 10y. Please complain to your distro

**Comment by JamesTheAwesomeDude** [comment]
@zarathustra19 for reference, [this was fixed by #15870](https://github.com/systemd/systemd/issues/15739#issuecomment-631617795)

Red Hat seems to be squarely to blame for its continued presence in their _latest release_, but [it looks like](https://ask.fedoraproject.org/t/very-slow-boot-waiting-for-dev-mapper-fedora-xxx/7524/3) there's a backported fix available while we wait.
